src/stepfunctions/steps/service.py

```diff
@@ -54,6 +47,18 @@ class EventBridgeApi(Enum):
     PutEvents = "putEvents"
 
 
+class GlueDataBrewApi(Enum):
+    StartJobRun = "startJobRun"
+
+
+class SnsApi(Enum):
+    Publish = "publish"
+
+
+class SqsApi(Enum):
+    SendMessage = "sendMessage"
+
```

Comment:
thanks for making this change! appreciate making it alphabetical.

in the future, try to keep these in a separate PR to maintain the focus and intent of this one 🙏

Reply:
Gotcha! will keep these seprate in the future :)